packages/cli/package.json

```diff
@@ -17,7 +17,7 @@
     "sap-cloud-sdk": "./bin/run"
   },
   "publishConfig": {
-    "access": "private"
+    "access": "restricted"
```

Comment:
Why? This should not be released yet and therefore stay private.

Reply:
According to the 
<img width="604" alt="Screenshot 2021-03-01 at 10 56 14" src="https://user-images.githubusercontent.com/54979414/109481186-bf5cde80-7a7c-11eb-9bdd-c0c7ce9161cf.png">

Reply:
I just saw this in my IDE and thought `private` doesn't work

Reply:
I think `restricted` is doing what `private` should be doing https://docs.npmjs.com/cli/v7/using-npm/config#config-settings

Reply:
I guess this is npm version 7 specific, but we are using yarn.
With yarn1, here is the [doc](https://classic.yarnpkg.com/en/docs/package-json/#toc-private), and I don't find it in the [migration guide](https://yarnpkg.com/getting-started/migration).

Reply:
I found the [yarn2 doc](https://yarnpkg.com/configuration/manifest#private).

Reply:
True that's a good catch!

Reply:
Actually, now that I look at it, we use the private flag in line 7, so I think the whole publishConfig is an npm artifact that is redundant/deprecated anyway